plugins/rsync/rsyncd_count

```diff
@@ -46,14 +46,14 @@ TEMP_FILE=`mktempfile munin-rsync-count.XXXXXX`
 
 if [ -n "$TEMP_FILE" -a -f "$TEMP_FILE" ]
 then
-	$LOGTAIL ${RSYNCD_LOG} $STATEFILE | grep ".* Total .* bytes\." > ${TEMP_FILE}
-	send=`grep ' send .* bytes' ${TEMP_FILE} | wc -l`
+	$LOGTAIL ${RSYNCD_LOG} $STATEFILE | grep ".* bytes .* total.*" > ${TEMP_FILE}
+	send=`grep ' sent .* bytes' ${TEMP_FILE} | wc -l`
```

Comment:
Why not just change `send=` to `sent=` as well for consistency? You'll also have to change it down below.